src/scmdata/run.py

```diff
@@ -715,6 +715,17 @@ def values(self) -> np.ndarray:
         """
         return np.asarray([ts._data.values for ts in self._ts])
 
+    @property
+    def is_empty(self) -> bool:
+        """
+        Is this :obj:`ScmRun` empty?
+
+        Returns
+        -------
+        bool
+        """
+        return np.equal(len(self._ts), 0)
```

Comment:
Why np.equal? len always returns an integer

Reply:
I read that comparing to zero (e.g. `len(self._ts) == 0`) is not the best idea so have been doing this instead.